job_templates/packaging_job.xml.em

```diff
@@ -35,6 +34,11 @@
             </a>
           </choices>
         </hudson.model.ChoiceParameterDefinition>
+        <hudson.model.StringParameterDefinition>
+          <name>CI_AMENT_BUILD_ARGS</name>
+          <description>Additional arguments passed to 'ament build'.</description>
+          <defaultValue>@(ament_build_args_default)</defaultValue>
+        </hudson.model.StringParameterDefinition>
```

Comment:
So here we can either move the CI_AMENT_BUILD_ARGS parameter inside our [common parameter snippet](https://github.com/ros2/ci/blob/master/job_templates/snippet/property_parameter-definition_common.xml.em) (given that after this PR all jobs will use it) or explicitly add it to the packaging job template. I'd prefer adding it to the template to avoid duplication. I was wondering if there is any objection before doing so